### ISSUE TITLE
Why does the console render the same information twice?

### ISSUE THREAD

**Issue by sichvoge** [mulesoft/api-console#321]
![image](https://cloud.githubusercontent.com/assets/498114/17966546/0f3ce01c-6aba-11e6-8b11-f59cb351a181.png)

RAML:

``` yaml
#%RAML 1.0
title: My API

/repository:
  /projects:
   post:
      description: Performs a Jira project create
      responses:
        200:
          description: Successful project creation
          body:
            application/json:
        400:
          description: User error
          body:
            application/json:
              example: !include BadKey.json
```

**Comment by Gregroam** [comment]
Yeah this is very ugly, especially for large responses

**Comment by paulcalabro** [comment]
One **workaround** that's easy to implement is to inject a style tag into the HTML to hide the erroneous DOM element. Here's a proof of concept that I got working:

```xml
	<flow name="api_router-console">
		<http:listener config-ref="http_listener_configuration" path="/console/*" doc:name="HTTP" />
		<apikit:console config-ref="api_router_configuration" doc:name="APIkit Console" />
		<choice doc:name="What is the Content Type?">
			<when expression="#[message.outboundProperties['Content-Type'] == 'text/html']">
				<byte-array-to-string-transformer doc:name="Transform the Byte Array to a String Object"
					encoding="UTF-8" mimeType="text/html" returnClass="java.lang.String" />
				<expression-transformer
					expression="#[message.payload.replaceAll('\\{disableRamlClientGenerator: true, disableThemeSwitcher: true\\}', '\\{disableTryIt: true\\}');]"
					doc:name="Transform the Message Payload to Disable the &quot;Try It&quot; Feature in the Console" />
				<expression-transformer
					expression="#[message.payload.replaceAll('&lt;/head&gt;', '&lt;style&gt;#example_app &gt; div &gt; div.raml-console-resource-panel-wrapper &gt; div.raml-console-resource-panel-primary.ng-scope &gt; div.ng-scope &gt; div.raml-console-resource-panel-primary-row.raml-console-resource-panel-content.raml-console-is-active.raml-console-response-container &gt; section &gt; div.raml-console-resource-response.ng-scope &gt; div &gt; div &gt; raml-body &gt; div &gt; div &gt; div.ng-isolate-scope &gt; div {display:none}&lt;/style&gt;\n&lt;/head&gt;');]"
					doc:name="Transform the Message Payload to Fix https://github.com/mulesoft/api-console/issues/321" />
			</when>
			<otherwise>
				<logger level="DEBUG" doc:name="Do Nothing" />
			</otherwise>
		</choice>
	</flow>

```

p.s. I also an modifying one of the API console flags on the fly as well. I left that in in case anyone else finds that useful.

**Comment by carowright** [comment]
Fix available in [v3.0.17](https://github.com/mulesoft/api-console/releases/tag/v3.0.17)
